pyproject.toml

```diff
@@ -11,7 +11,7 @@ classifiers = [
 
 [tool.poetry]
 name = "tidal_per_transformers"
-version = "0.2.8"
+version = "0.3.0"
```

Comment:
This is a pretty major change, even if it's backward compatible, the behaviour is not the same, bumping the minor